core/src/main/java/org/testcontainers/containers/output/ToStringConsumer.java

```diff
@@ -10,21 +10,14 @@
  * Created by rnorth on 26/03/2016.
  */
 public class ToStringConsumer extends BaseConsumer<ToStringConsumer> {
-    private static final byte[] NEW_LINE = "\n".getBytes();
-
-    private boolean firstLine = true;
     private ByteArrayOutputStream stringBuffer = new ByteArrayOutputStream();
 
     @Override
     public void accept(OutputFrame outputFrame) {
         try {
             if (outputFrame.getBytes() != null) {
-                if (!firstLine) {
-                    stringBuffer.write(NEW_LINE);
-                }
                 stringBuffer.write(outputFrame.getBytes());
                 stringBuffer.flush();
-                firstLine = false;
```

Comment:
This essentially reverts the change done here: https://github.com/testcontainers/testcontainers-java/pull/643/files#diff-321431024ed34199142ebf2d70a46ba89f8a09e66582d40527885123e429b4d9R22-R24

It's not entirely clear to me if this will work for all use cases; it could very well be that this breaks (so it might be that we'll have to conditionalize it). I was unable to run all the tests locally, so maybe CI will shed some light on this.